### ISSUE TITLE
Problems with citing in German

### ISSUE THREAD

**Issue by UFOMelkor** [Zettlr/Zettlr#3948]
### Description

I have a problem with the usage of locators when citing and switching the language into German. I'm pretty sure that I do something wrong. When I export the following file as PDF, I get the expected output.

```markdown
---
lang: en
---

- *40*: [@Beck2003, 40]
- *p. 40:* [@Beck2003, p. 40]
- *S. 40:* [@Beck2003, S. 40]
- *chap. 40:* [@Beck2003, chap. 40]
- *Kap. 40:* [@Beck2003, Kap. 40]
```

![image](https://user-images.githubusercontent.com/571106/203832831-bf87775c-02e4-4e28-850b-74df956a4a01.png)

When I switch the language in the YAML frontmatters to `de` I get the following result.

![image](https://user-images.githubusercontent.com/571106/203833280-476a731c-7a26-47b8-8697-519142b6d884.png)

Neither the English locators nor the German locators are recognized. The preview, however, recognizes all locators:

![image](https://user-images.githubusercontent.com/571106/203833513-53767527-d4eb-4b61-a703-e716a35286d6.png)

### Reproducing

Have a citation database with the cite key `@Beck2003` (or replace it in the file with another one) and create a file with the following content:

Have 

```markdown
---
lang: de
---

- *40*: [@Beck2003, 40]
- *p. 40:* [@Beck2003, p. 40]
- *S. 40:* [@Beck2003, S. 40]
- *chap. 40:* [@Beck2003, chap. 40]
- *Kap. 40:* [@Beck2003, Kap. 40]
```

Then export the file (as PDF Document or PDF (LaTeX)) and look.

### Zettlr Version

Stable (most recent version)

### Specify version

2.3.0

### Installation Method

From the Website or GitHub

### Your Platform

- [ ] Windows
- [ ] macOS
- [X] Linux

### Architecture

- [ ] Intel 64bit
- [ ] ARM 64bit

### Operating System Version

Ubuntu 22.04

**Comment by nathanlesage** [comment]
What CSL style are you using? None (a.k.a. the default)? Or a specific one? This could also indicate a problem with the Pandoc version that has been bundled with Pandoc. This problem could potentially still exist. Could you try the following steps and report back if that solved your problem?

0. Before you do anything, try to export using [a few different CSL styles](https://www.zotero.org/styles), if that then works, the issue is with the CSL style you're using and you can ignore all other points in this list
1. Install the [most recent Pandoc version](https://pandoc.org/installing.html) manually on your system according to the setup instructions
2. In the Zettlr preferences, head over to the Export tab and uncheck "Use internal Pandoc for exports"
3. Restart Zettlr for that change to take effect
4. Try exporting again

If that then works, this indicates that this was a bug in Pandoc that has been fixed in the meantime. If so, you can keep using the system-wide Pandoc install and wait for me to release Zettlr 3.0 (possibly Q1 or Q2 of 2023).

If that doesn't work, that would indicate that there's a Pandoc related issue, then I can go and ping their team to have a look at this issue.

**Comment by UFOMelkor** [comment]
Thank you very much, it was indeed the default Pandoc version shipped by Ubuntu 2022.04 which has an error.
